hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/7_8_0/6603-combo-index-sp-not-invoked-with-dateparam.yaml

```
@@ -0,0 +1,6 @@
+---
+type: fix
+issue: 6603
+jira: SMILE-8048
+title: "Previously, searches where processing was optimised with a combo index search parameters would skip searching the 
+optimised indexes when the query string included date parameters. This issue is fixed."
```

I'm assuming based on what was fixed that the issue is actually when the query string contained a date range (i.e. multiple repetitions of the same date parameter). Might be helpful to note that.

```suggestion
optimised indexes when the query string included a date range (i.e. multiple instances of the same date parameter). This issue is fixed."
```

---

updated the changelog to include the recommended syntax.